test/e2e/storage/persistent_volumes.go

```diff
@@ -313,6 +313,30 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				pvc = nil
 			})
 		})
+
+		ginkgo.Context("delete pv prior", func() {
+			ginkgo.AfterEach(func() {
+				framework.Logf("AfterEach: Cleaning up test resources.")
+				if errs := e2epv.PVPVCCleanup(c, ns, pv, pvc); len(errs) > 0 {
```

Comment:
This function PVPVCCleanup deletes PVC first.
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/pv/pv.go#L146-L155

You need to write a function that deletes PV first.  Switching the order will have different results.